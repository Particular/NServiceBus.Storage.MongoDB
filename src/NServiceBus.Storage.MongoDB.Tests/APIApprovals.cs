namespace NServiceBus.Storage.MongoDB.Tests
{
    using NUnit.Framework;
    using Particular.Approvals;
    using PublicApiGenerator;

    [TestFixture]
    public class APIApprovals
    {
        [Test]
        public void Approve()
        {
            var publicApi = ApiGenerator.GeneratePublicApi(typeof(MongoPersistence).Assembly, excludeAttributes: new[] {"System.Runtime.Versioning.TargetFrameworkAttribute"});
            Approver.Verify(publicApi);
        }
    }
}