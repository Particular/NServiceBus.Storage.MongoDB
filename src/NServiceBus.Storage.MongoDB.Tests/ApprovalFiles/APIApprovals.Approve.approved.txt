[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("NServiceBus.Storage.MongoDB.Tests")]
namespace NServiceBus
{
    public class MongoDBPersistence : NServiceBus.Persistence.PersistenceDefinition
    {
        public MongoDBPersistence() { }
    }
    public class static MongoDBSettingsExtensions
    {
        public static NServiceBus.PersistenceExtensions<NServiceBus.MongoDBPersistence> ConnectionString(this NServiceBus.PersistenceExtensions<NServiceBus.MongoDBPersistence> persistenceExtensions, string connectionString) { }
        public static NServiceBus.PersistenceExtensions<NServiceBus.MongoDBPersistence> VersionFieldName(this NServiceBus.PersistenceExtensions<NServiceBus.MongoDBPersistence> persistenceExtensions, string versionFieldnName) { }
    }
}